dask/bytes/tests/test_http.py

```diff
@@ -6,6 +6,7 @@
 import fsspec
 import pytest
 from fsspec.core import open_files
+from packaging.version import parse
 from packaging.version import parse as parse_version
```

Comment:
Looks like it's already imported as `parse_version` :)

Reply:
d'oh